### PR TITLE
Specified webhook service namespace, updated default image, and added support for existing namespace

### PR DESCRIPTION
**Pull request by yuchaoran2011**
Addressed this [issue](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/issues/290). Now specify the namespace in which the webhook service is launched.

### CODE REVIEW

**Comment by liyinan926** [comment]
LGTM. @foxish for approval.

**Review by aditanase — APPROVED** [review]

**Comment by aditanase** [comment]
Just tested this with helm local install, it works.

**Comment by yuchaoran2011** [comment]
Pinging @foxish

**Comment by liyinan926** [comment]
@yuchaoran2011 wondering can we change the default image to `gcr.io/spark-operator/spark-operator:v2.3.1-v1alpha1-latest` as part of this PR? This makes it consistent with what's used in the operator repo and also makes it easier for me to push image update when there's bug fixes or new features. Thanks!

**Comment by yuchaoran2011** [comment]
@liyinan926 No problem. Done.

**Comment by yuchaoran2011** [comment]
@liyinan926 I just pushed a new commit to the PR. 

Previously, the chart created a new namespace to install the operator no matter what. Now the chart supports installing the operator in an existing namespace. 

The chart used to set up Spark RBAC in the `default` namespace. Therefore Spark jobs deployed in other namespaces would fail because `roles` and `rolebindings` are namespaced resources. Now the chart gives the user the ability to choose the namespace they want to set up RBAC in. (It defaults to `default`).

**Comment by aditanase** [comment]
@yuchaoran2011 great addition, thanks! I was just chatting with a colleague about more constrained K8S deployments, where the ops team might restrict access to e.g. 1 namespace per team.

**Comment by yuchaoran2011** [comment]
@aditanase Sure! We ran into the same use case and thought others might need this feature too.

**Comment by yuchaoran2011** [comment]
@liyinan926 I added the `OWNERS` file with your and my Github usernames. I'm following steps [here](https://github.com/helm/charts/blob/master/README.md#owning-and-maintaining-a-chart), hoping we can get merge privileges on this chart. This way, changes can be incorporated much faster in the future.

File: incubator/sparkoperator/values.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
-operatorImageName: lightbend/sparkoperator
-operatorVersion: 2.3.1
+operatorImageName: gcr.io/spark-operator/spark-operator
+operatorVersion: v2.3.1-v1alpha1-latest
```
Comment (foxish) [inline incubator/sparkoperator/values.yaml:2]: please confirm that this is the version you want to make available.

**Review by foxish — APPROVED** [review]

File: incubator/sparkoperator/values.yaml (hunk 1 above)
Comment (yuchaoran2011) [reply incubator/sparkoperator/values.yaml:2]: Yes it is for now. When Spark 2.4 is out in a few weeks, I'll make another PR to update the image to use it.

**Pull request merged**